scripts/create-nuxt-app.sh

```diff
@@ -10,7 +10,7 @@ source "$script_dir/global/functions.sh"
 
 title='Create Nuxt App'
 
-export ANSWERS=$(cat <<'EOF'
+export ANSWERS=$(tr -d "\n" <<'EOF'
```

Comment:
I just realized after viewing the "Notes" section of the generated README that we should revert this change, but change this line below:

```sh
npx create-nuxt-app $dir --answers "$ANSWERS"
```

To this:

```sh
npx create-nuxt-app $dir --answers "${ANSWERS//$'\n'/}"
```

So that the line breaks are only removed when `$ANSWERS` is passed to the `create-nuxt-app` command.